packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -85,13 +122,89 @@ class Tooltip extends ThemePropertyMixin(ElementMixin(PolymerElement)) {
   /** @private */
   __targetChanged(target, oldTarget) {
     if (oldTarget) {
+      oldTarget.removeEventListener('mouseenter', this.__boundOnMouseEnter);
+      oldTarget.removeEventListener('mouseleave', this.__boundOnMouseLeave);
+      oldTarget.removeEventListener('focusin', this.__boundOnFocusin);
+      oldTarget.removeEventListener('focusout', this.__boundOnFocusout);
+      oldTarget.removeEventListener('mousedown', this.__boundOnMouseDown);
```

Comment:
nit: these could also use a test case (changing the target dynamically is relevant in grid)

Reply:
Thanks, added tests.